sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/EventGridClientOptions.cs

```diff
@@ -36,6 +36,11 @@ public enum ServiceVersion
         /// </summary>
         internal ServiceVersion Version { get; }
 
+        /// <summary>
+        /// Used to serialize the given events to JSON.
```

Comment:
We might want to say UTF-8 Encoded JSON here, since if you return the encoded JSON in some other string encoding, I think we'll fail when we use `JsonDocument.Parse` later.